dev/tools/gen_keycodes/data/README.md

```diff
@@ -0,0 +1,82 @@
+## Files
+
+### General
+
+| File name | Explanation |
+| ---- | ---- |
+| [`physical_key_data.json`](dev/tools/gen_keycodes/data/physical_key_data.json) | Contains the merged physical key data from all the other sources. This file is regenerated if "--collect" is specified for the gen_keycodes script, or used as a source otherwise. |
```

Comment:
I think you can just use:
```
[`physical_key_data.json`](physical_key_data.json)
```

Or does that not work?

Reply:
Yes you're right. In fact, your way is the only one that works instead of mine.